### PR TITLE
Make VA information checkboxes display conditionally

### PR DESCRIPTION
**Pull request by joesoftware**
Surprisingly simple change to make checkboxes conditional for VA Information section.

### CODE REVIEW

File: _health-care/_js/components/personal-information/VAInformationSection.jsx [hunk 1]
```diff
@@ -17,17 +17,29 @@ class VaInformationSection extends React.Component {
           <ErrorableCheckbox
               label="Are you VA Service Connected 50% to 100% Disabled?"
               checked={this.props.data.isVaServiceConnected}
+              required={!false}
```
Comment (akainic) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: Why include this?

File: _health-care/_js/components/personal-information/VAInformationSection.jsx [hunk 2]
```diff
@@ -17,17 +17,29 @@ class VaInformationSection extends React.Component {
...
+                checked={this.props.data.compensableVaServiceConnected}
+                required={!false}
+                onValueChange={(update) => {this.props.onStateChange('compensableVaServiceConnected', update);}}/>
+          }
 
-          <ErrorableCheckbox
-              label="Are you compensable VA Service Connected 0% - 40%?"
-              checked={this.props.data.compensableVaServiceConnected}
-              onValueChange={(update) => {this.props.onStateChange('compensableVaServiceConnected', update);}}/>
+          {this.props.data.isVaServiceConnected === true &&
```
Comment (akainic) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: If this paragraph is only dependent `isVaServiceConnected` you can probably include it in the above check.

File: _health-care/_js/components/personal-information/VAInformationSection.jsx (hunk 1 above)
Comment (joesoftware) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: This is a required field... assumed this was the proper usage.

File: _health-care/_js/components/personal-information/VAInformationSection.jsx (hunk 2 above)
Comment (joesoftware) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: When I tried to consolidate two components inside the same bracket block, the linter complained.

File: _health-care/_js/components/personal-information/VAInformationSection.jsx (hunk 1 above)
Comment (akainic) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: I don't think it's a required field. That wouldn't make sense, what would you do if your answer to this is false? You have no choice but to not check it.

Comment (joesoftware) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: I understand your question now. The only rational for making this a required field would be somehow "force" the Vet to acknowledge that she'd seen the information. I'll remove the attribute.

File: _health-care/_js/components/personal-information/VAInformationSection.jsx (hunk 2 above)
Comment (joesoftware) [inline _health-care/_js/components/personal-information/VAInformationSection.jsx]: I see now how do combine components using `<div>`s. Will change.

**Comment by akainic** [comment]
LGTM! The next step is to then hide the financial questions if they receive a pension, correct? For that we might want to display a message on the financial section pages that says something like "You are not required to complete this information because you indicated that you receive a VA pension".

**Pull request merged**